src/utils/getMetaData.js

```diff
@@ -53,7 +54,8 @@ function getMetaData(page) {
     "navigationBackgroundImage",
     "image",
   ]);
-  if (tcImage) {
+
+  if (tcImage && !isVideoObj(page.get("navigationBackgroundImage"))) {
```

Comment:
This is not how it should be. `tcImage` is an Url not a Obj.

How about this? We revert the changes of `isVideoObj` in this file and do the following refactoring:

Rename `firstUrlForAttributes` to `firstImageUrlForAttributes` and extend it to this:

```js
function firstImageUrlForAttributes(obj, attributes) {
  let url;

  attributes.forEach(attribute => {
    if (url || isVideoObj(obj)) {
      return;
    }
    const binary = obj.get(attribute);
    url = urlFromBinary(binary);
  });

  return url;
}
```